contracts/trade-shield-contract/src/action/sudo/process_orders.rs

```diff
@@ -519,6 +559,38 @@ fn _calculate_token_out_min_amount(_order: &SpotOrder) -> Int128 {
     Int128::zero()
 }
 
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use cosmwasm_std::testing::MockStorage;
+
+    use super::*;
+
+    #[test]
+    fn test_split_function() {
+        let ids = vec![0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+        let mut n_spot_order = Some(5);
+        let closest_index = 12;
+        let order_type = SpotOrderType::LimitSell;
+        let market_price = Decimal::from_str("0.1").unwrap();
+        let mut mock_storage = MockStorage::new();
+
+        let result = split_spot_order(
+            closest_index,
+            &mut n_spot_order,
+            order_type,
+            market_price,
+            ids,
+            &mut mock_storage,
+        )
+        .unwrap();
+
+        assert_eq!(result, vec![0, 1, 2, 3, 4]);
+        assert_eq!(n_spot_order, Some(0));
+    }
```

Comment:
Is it possible to add test for `process_orders` with `n_perptual_order`/`n_spot_order` set ?

Reply:
here the [tests](https://github.com/elys-network/bindings/pull/356/files#diff-adec7bdd86c396a9c041787829103abedd8c049dd3972c7fc26dd63c4327899b) that show how process order behave with a limit

Reply:
thanks, I have one question, added below